docker/README.md

```diff
@@ -0,0 +1,7 @@
+# Configuration
+
+The following environment variables are supported:
+
+- `PUPPERWARE_DISABLE_ANALYTICS`
+
+  Set to `true` to disable Google Analytics metrics.
```

Comment:
I'm not looking to properly document these containers right now. We can add a Trello card for that and circle back.

Just wanted to get the automation/plumbing in place, and start the README with the ENV configuration values.

Although, we already have some documentation up on DockerHub for (puppetdb)[https://hub.docker.com/r/puppet/puppetdb] and (puppetserver)[https://hub.docker.com/r/puppet/puppetserver], so I might include that text in the new README.md's for those repos.